key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Yeah, it doesn't matter because I'll delete before merge—we'll get a real key when upon publish.